tests/serializers/test_natural.py

```diff
@@ -69,6 +69,27 @@ def natural_key_test(format, self):
     self.assertIsNone(books[1].object.pk)
 
 
+def natural_primary_keys_multi_table_test(format, self):
```

Comment:
It would be nice to add a docstring that explains the expected behavior that's being tested.